app/models/managed_interactive.rb

```diff
@@ -189,13 +190,17 @@ def to_interactive
   end
 
   def duplicate
+    # Remove linked_interactives from the hash since it can't be mapped to a database column like the other 
+    # properties in the hash can, and so causes an error when we try to create the duplicate interactive.
     new_interactive_hash = self.to_hash.except!(:linked_interactives)
     # Generate a new object with those values
     ManagedInteractive.new(new_interactive_hash)
     # N.B. the duplicate hasn't been saved yet
   end
 
   def export
+    # Remove linked_interactives from the hash since it can't be mapped to a database column like the other 
+    # properties in the hash can, and so would cause an error on import.
```

Comment:
I don't think this comment is correct. As are as I can tell from the code the the export already includes linked_interactives. It seems to be added here: https://github.com/concord-consortium/lara/blob/b4bf12d4411cdbd7eacd09117640b658a3e94fac/app/services/lara_serialization_helper.rb#L25-L35 

So the problem is not the import. I haven't checked, but I'd guess the format the linked_interactives array added by the lara serialization helper is different than the format being added by to_hash

Reply:
@scytacki Thanks. I've updated the comment. 

It seems like importing is simply broken for an unrelated reason and I somehow missed that. I checked on lara-demo.concordqa.org and confirmed importing is broken there. I'll make a separate story about that.

Reply:
(There's actually [a story about importing already](https://www.pivotaltracker.com/story/show/179237412).)